src/store/domain/messagesView/actions.ts

```diff
@@ -41,7 +45,9 @@ export const actions = defineActions({
       commit.setIsReachedEnd(true)
     }
     commit.setCurrentOffset(state.currentOffset + state.fetchLimit)
-    const messageIds = messages.map((message: Message) => message.id ?? '')
+    const messageIds: MessageId[] = messages.map(
+      (message: Message) => message.id ?? ''
+    )
```

Comment:
```suggestion
    const messageIds: MessageId[] = messages.map(message => message.id)
```
node-traqのバージョンあげてundefinedになることはなくしたので今はこう書けるようになりました